docs/src/content/docs/book/message-mode.mdx

```diff
@@ -11,7 +13,7 @@ It's possible to use raw [`Int{:tact}`][int] values and manually provide them fo
 
 Mode value | Constant name                 | Description
 ---------: | :---------------------------- | -----------
-$0$        | -                             | Ordinary message (default).
+$0$        | <Badge text="Since Tact 1.6" variant="tip"/> `SendDefaultMode{:tact}` | Ordinary message (default).
```

Comment:
maybe move the badge into Description?

Reply:
We've discussed that a bit with jubnzv, and my position is that the mode itself was present there all along while only the constant name for it is new. And that the table layout in the source matters less since this table isn't going to grow big and user-facing UX matters more